packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -249,12 +270,13 @@ export class MonitorATMServicesViewImpl extends React.PureComponent<TProps, Stat
                 metrics.serviceError.service_latencies_95
               }
               loading={metrics.loading}
-              name="Latency, ms"
+              name={`Latency, ${convertTimeUnitToShortTerm(displayTimeUnit)}`}
```

Comment:
I prefer putting the units in brackets like "Latency (ms)" or "Latency (min)", instead of a comma. 

What do you/others think?

Reply:
done